css/main.css

```diff
@@ -0,0 +1,75 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 700;
+}
+
+::-webkit-scrollbar {
+  width: 6px;
+  height: 10px;
+  display: none;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.logo > img {
+  width: 80px;
+  height: 80px;
+  padding: 10px;
+}
```

Comment:
- [ ] Kyndly adjust your nav-bar, your logo image is too big an makes you nav-bar take more space than the one showed on the figma design, also notice that the burguer menu in the figma design is not that big and has a blue color (#6070FF).
### Figma
![image](https://user-images.githubusercontent.com/95243996/191730954-aac5e385-1b33-4046-b6cf-ed3df7715e56.png)
 
### Your design 
![image](https://user-images.githubusercontent.com/95243996/191732479-5e7acbf5-d26b-4593-8b25-61690dab7ec5.png)